src/groups/mqb/mqbc/mqbc_clusterdata.h

```diff
@@ -79,13 +79,13 @@ class ClusterDataIdentity {
   private:
     // DATA
 
-    bsl::string d_name;
+    const bsl::string d_name;
```

Comment:
I would avoid adding these `const`s here, this can lead to unusually restrictive class design issues later. If the intent is to ensure these fields are never modified after class construction, then we should provide an interface that guarantees that.

Reply:
Good point. Our interface already has such guarantees, so I will remove the `const`s.

Reply:
d